### DISCUSSION TITLE
续费后变成免费版，无法升级请退款

### DISCUSSION THREAD

**Discussion by Liam983552** [getlantern/lantern-forum-cn#782] [General]
邮箱 fujoyabc@hotmail.com

到期前已续费，过几天就变成了免费版，无法使用邮箱授权为专业版，交易信息如下，烦请退款

![Screenshot_2024-08-13-19-21-28-255_com cmbchina ccd pluto cmbActivity](https://github.com/user-attachments/assets/9ecd1525-a8db-48df-b8bc-99f9dcbcde0a)

**Comment by Liam983552** [comment]
@Mike777Khoo   提交问题好几天了，请给个回复

**Reply by MaxMKMax** [reply]
@Liam983552 您好. 马上给您安排. 不好意思, 可能漏掉您的信息了.

**Reply by MaxMKMax** [reply]
@Liam983552 
我们帮您升级, 帅哥别退款好不好? 我再帮您多拿几个月免费.

**Comment by xilijuli** [comment]
我也是这样的问题，账号是345408353@qq.com。两年多的钱。

**Reply by MaxMKMax** [reply]
@xilijuli 您好. 请您提供我支付单据和邮箱. 我让客服帮您升级.

**Comment by Liam983552** [comment]
这么多人出现同样的问题，要等到什么时候才有回复，没有解决方案，没有进度公告，连安抚都做不到，真的是把大家的信任耗光了

**Reply by MaxMKMax** [reply]
@Liam983552 真的非常抱歉. 因为太多人遇到问题了, 一时之间无法安抚所有人. 我也有工作时间限制的. 
受公司限制不能超过特定时间. 所以不能每个细节都照顾好. 希望您能多多包涵. 谢谢您能够体谅我.
再向您说声抱歉! 也谢谢您支持蓝灯!

**Comment by Pro-D** [comment]
同样的问题，需要申请退款，始终是免费版本，已经交了1年的钱

**Reply by MaxMKMax** [reply]
@Pro-D 您好. 在另一个帖子已经回您了.

**Comment by Liam983552** [comment]
如能升级就烦请升级为专业版，不好升级就请退款

升级或退款都可以，只希望尽快解决

谢谢！

________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年8月27日 0:30
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: Liam983552 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 续费后变成免费版，无法升级请退款 (Discussion #782)


@Liam983552<https://github.com/Liam983552>
我们帮您升级, 帅哥别退款好不好? 我再帮您多拿几个月免费.

―

**Reply by MaxMKMax** [reply]
@Liam983552 您好. 真的非常抱歉让您操心了. 我们会尽快给您解决问题. 我们会帮您升级专业版. 再给我们 1-2 天时间. 谢谢您.

**Comment by MaxMKMax** [comment]
@Liam983552 真的非常抱歉. 因为太多人遇到问题了, 一时之间无法安抚所有人. 我也有工作时间限制的.
受公司限制不能超过特定时间. 所以不能每个细节都照顾好. 希望您能多多包涵. 谢谢您能够体谅我.
再向您说声抱歉! 也谢谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
![image](https://github.com/user-attachments/assets/8aceb50d-7187-4104-ab11-b0d4de09a623)

@Liam983552 晚上好. 问题已经解决. 给您升级好了! 不好意思让您久等了. 谢谢您支持蓝灯!

**Reply by Liam983552** [reply]
非常感谢，希望与我有同样问题的也可以帮忙尽快解决，大家都是抱有同样信任充值的，不要让任何一个人失望
